jira/server/scio_search/src/main/java/ScioSearchConfigRestPlugin/impl/ScioPermissionScheme.java

```diff
@@ -0,0 +1,86 @@
+package ScioSearchConfigRestPlugin.impl;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.permission.PermissionSchemeManager;
+import com.atlassian.jira.project.Project;
+import com.atlassian.jira.scheme.Scheme;
+import com.atlassian.jira.scheme.SchemeEntity;
+import com.atlassian.jira.security.roles.ProjectRole;
+import com.atlassian.jira.security.roles.ProjectRoleManager;
+import com.atlassian.jira.user.ApplicationUser;
+import com.atlassian.plugin.spring.scanner.annotation.imports.JiraImport;
+import com.atlassian.sal.api.user.UserManager;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+
+@Named
+@Path("/permission_scheme")
+public class ScioPermissionScheme {
+    @JiraImport
+    private final UserManager userManager;
+    @Inject
+    public ScioPermissionScheme(UserManager userManager) {
+
+        this.userManager = userManager;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public PermissionSchemeResponse getIssueSecuritySchemeMembers(@QueryParam("projectId") String projectId) {
+        Utils.validateUserIsAdmin(userManager);
+
+        Project project = ComponentAccessor.getProjectManager().getProjectObj(Long.parseLong(projectId));
+        Scheme scheme = ComponentAccessor.getComponentOfType(PermissionSchemeManager.class).getSchemeFor(project);
+        PermissionSchemeResponse response = new PermissionSchemeResponse();
+        if (scheme == null) {
+            return response;
+        }
+        response.id = scheme.getId().toString();
+        response.name = scheme.getName();
+        response.description = scheme.getDescription();
+        response.permissions = new ArrayList<>();
+        for (SchemeEntity entity : scheme.getEntities()) {
```

Comment:
were you able to test this? If so, might be good to update [this](https://askscio.atlassian.net/wiki/spaces/ENGINEERIN/pages/1818624063/Plugin+development) for Jira